hack/boilerplate/boilerplate.py

```diff
@@ -218,7 +218,7 @@ def get_regexs():
     regexs["date"] = re.compile(get_dates())
     # strip // +build \n\n build constraints
     regexs["go_build_constraints"] = re.compile(
-        r"^(// \+build.*\n)+\n", re.MULTILINE)
+        r"^(// \+build.*\n|//go:build.*\n)+\n", re.MULTILINE)
```

Comment:
It looks like this line was fixed in your another PR https://github.com/kubernetes/kubernetes/pull/100361, revert ?

Reply:
no, this just fixed make verify

Reply:
sry, exist an redundancy commit